commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -108,7 +109,7 @@ public void testGetStringResourceByName_returnsDefaultValue() {
 
         String stringResource = TransferMethodUtils.getStringResourceByName(mContext,
                 "some type");
-        assertThat(stringResource, is("bank acc"));
+        MatcherAssert.assertThat(stringResource, Matchers.is("bank acc"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
remove